crates/client/sync/src/fetch/fetchers.rs

```diff
@@ -40,15 +41,25 @@ pub struct FetchConfig {
     pub sync_polling_interval: Option<Duration>,
     /// Number of blocks to sync (for testing purposes).
     pub n_blocks_to_sync: Option<u64>,
+    /// Number of blocks between db flushes
+    pub flush_every_n_blocks: u16,
     /// Stops the node once all blocks have been synced (for testing purposes)
     pub stop_on_sync: bool,
+    /// Number of blocks to fetch in parallel during the sync process
+    pub sync_parallelism: u8,
+    /// True if the node is called with `--warp-update-receiver`
+    pub warp_update: bool,
+    /// The port used for nodes to make rpc calls during a warp update.
+    pub warp_update_port_rpc: u16,
```

Comment:
for info, ports are u16 because at the os level they are u16 - the os accepts an u16